src/win32/mouse.c

```diff
@@ -20,13 +20,30 @@
 	: ((button) == RIGHT_BUTTON ? MOUSEEVENTF_RIGHTDOWN \
 	: MOUSEEVENTF_MIDDLEDOWN))
 
+int CalculateAbsoluteCoordinateX(int x) {
+	MMSize displaySize = getMainDisplaySize();
+	return (x * 65536) / displaySize.width;
```

Comment:
**Nitpick:** I would rephrase the calculation to `(x / displaySize.width) * 65536` and `(y / displaySize.height) * 65536`, but that's just my personal tase and due to how I read the equation.